mlflow/genai/labeling/__init__.py

```diff
@@ -77,12 +77,12 @@ def get_labeling_session(name: str) -> LabelingSession:
         (
             labeling_session
             for labeling_session in labeling_sessions
-            if labeling_session.name == name
+            if labeling_session.mlflow_run_id == run_id
```

Comment:
It would seem more efficient for us to do a `get_run()` here and then convert the response into a labeling session object. Any concerns with that approach?

Reply:
Can we look at the implementation of `get_labeling_sessions()`?

Reply:
@dbczumar `get_labeling_sessions()` is a list endpoint. We have a `GetLabelingSession` endpoint, but it takes `labeling_session_id`, not `run_id`. Do you think this current implementation is sufficient and as a follow-up, we can swap to a the `GetLabelingSession` endpoint once it takes a run id?